modules/newrelic/settings.go

```diff
@@ -12,19 +12,19 @@ const defaultTitle = "NewRelic"
 type Settings struct {
 	common *cfg.Common
 
-	apiKey        string
-	applicationID int
-	deployCount   int
+	apiKey         string
+	deployCount    int
+	applicationIDs []interface{}
 }
 
 func NewSettingsFromYAML(name string, ymlConfig *config.Config, globalConfig *config.Config) *Settings {
 
 	settings := Settings{
 		common: cfg.NewCommonSettingsFromModule(name, defaultTitle, ymlConfig, globalConfig),
 
-		apiKey:        ymlConfig.UString("apiKey", os.Getenv("WTF_NEW_RELIC_API_KEY")),
-		applicationID: ymlConfig.UInt("applicationID"),
-		deployCount:   ymlConfig.UInt("deployCount", 5),
+		apiKey:         ymlConfig.UString("apiKey", os.Getenv("WTF_NEW_RELIC_API_KEY")),
+		deployCount:    ymlConfig.UInt("deployCount", 5),
+		applicationIDs: ymlConfig.UList("applicationIDs"),
```

Comment:
You could keep applicationID for a non-breaking change. Either by appending it to applicationIDs, or doing something like the `arrayifyProjects` in the jira settings file